### PR TITLE
[AS-31] Format and configure Projects screen

### PR DESCRIPTION
**Pull request by andrmdmd**

### CODE REVIEW

File: src/components/projectTile/ProjectTile.css
Comment (mslup) [inline src/components/projectTile/ProjectTile.css:1]: nitpick: could you fix indentation in this file

Comment (mslup) [inline src/components/projectTile/ProjectTile.css:1]: nitpick: we need to decide on the convention of casing css class names. as of now most of class names are in camelCase, but we can switch to kebab-case if you prefer it so (but then I'd like to have a Jira ticket for changing them everywhere)

File: src/components/projectTile/ProjectTile.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  project: Project;
+  onClick: () => void;
+}
+
+const ProjectTile = ({ project, onClick }: ProjectTileProps): JSX.Element => (
+  <Card className="project-card">
+    <div onClick={onClick} role="button" tabIndex={0} className="project-wrapper">
+      <img
+        className="project-thumbnail"
+        src={project.thumb || placeholder}
```
Comment (mslup) [inline src/components/projectTile/ProjectTile.tsx:16]: that's probably out of scope of this PR, but: if thumbnail URL is provided, but invalid (cannot fetch an image), the image changes its size and the whole tile gets smaller

File: src/components/scrollbar/ScrollBar.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+const ScrollBar = ({
+  children,
+  className = '',
+}: ScrollBarProps): JSX.Element => (
+  <Scrollbars
+    className={className}
+    renderTrackHorizontal={() => <div style={{ display: 'none' }} />}
+    renderThumbVertical={({ style, ...props }) => (
+      <div
+        // eslint-disable-next-line react/jsx-props-no-spreading
```
Comment (mslup) [inline src/components/scrollbar/ScrollBar.tsx]: can we add this to global ESLint configuration? I've disabled this rule a couple of times already as well, I don't see a reason to keep it.

File: src/components/scrollbar/ScrollBar.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+          backgroundColor: 'rgba(29, 108, 226, 0.27)',
+          borderRadius: 10,
+        }}
+      />
+    )}
+    renderTrackVertical={({ style, ...props }) => (
+      <div
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        style={{
```
Comment (mslup) [inline src/components/scrollbar/ScrollBar.tsx]: could we move it to the CSS file 🙏

File: src/types/Scene.tsx [hunk 4]
```diff
@@ -38,6 +38,10 @@ export interface Project {
   latitude: number;
   longitude: number;
   orientation: number;
+  createdAt: number;
+  modifiedAt: number;
+  thumb: string;
+  id?: string;
```
Comment (mslup) [inline src/types/Scene.tsx:44]: why '?'?

File: src/utils/firebaseUtils.tsx [hunk 5]
```diff
@@ -45,6 +44,19 @@ export const getProject = async (
...
+export const getAllProjects = async (userId: string): Promise<Project[]> => {
+  const projectsRef = collection(db, 'users', userId, 'projects');
+  const snapshot = await getDocs(projectsRef);
+
+  const projects: Project[] = [];
+  snapshot.forEach((doc) => {
+    const data = doc.data() as Project;
+    projects.push({ ...data, id: doc.id });
+  });
+  console.log(projects);
```
Comment (mslup) [inline src/utils/firebaseUtils.tsx]:
```suggestion

```

**Review by mslup — CHANGES_REQUESTED** [review]
great job overall! left some minor comments.

File: src/components/projectTile/ProjectTile.css
Comment (andrmdmd) [reply src/components/projectTile/ProjectTile.css:1]: What do you mean by that? This file is indented correctly based on my linter

File: src/types/Scene.tsx (hunk 4 above)
Comment (andrmdmd) [reply src/types/Scene.tsx:44]: `id` is related to database only and cannot be declared when creating the project to be added to the database

**Review by mslup — DISMISSED** [review]

**Comment by mslup** [comment]
fenomenal

**Review by mslup — APPROVED** [review]

**Pull request merged**